### DISCUSSION TITLE
How to get all known forms of an English word?

### DISCUSSION THREAD

**Discussion by monocongo** [JohnSnowLabs/spark-nlp#11543] [Q&A]
I am porting pandas/NLTK code to Spark NLP. One of the packages used in the original code is `word_forms` which provides the function [`get_word_forms`](https://github.com/gutfeeling/word_forms/blob/master/word_forms/word_forms.py#L75) that gets all forms of an English word. Is there an equivalent function in Spark NLP, or a good way to create one?

Thanks in advance for any suggestions or advice.

**Comment by maziyarpanahi** [comment]
An interesting feature, I think it's a reverse Lemmatization. There was a feature request to get all the possible lemmatization from a Lemmatizer pre-trained model, this can be also a new function in Lemmatizer that simply giving a `word` will return all the other existing forms.

This feature currently doesn't exist, but I will look to see how plausible it is at this time and get back to you
